src/runmode-dpdk.c

```diff
@@ -616,6 +618,14 @@ static int ConfigSetChecksumOffload(DPDKIfaceConfig *iconf, int entry_bool)
     SCReturnInt(0);
 }
 
+static int ConfigSetVlanStrip(DPDKIfaceConfig *iconf, int entry_bool)
+{
+    SCEnter();
```

Comment:
This function can be void returning. The macros to use then are `SCEnter();` and `SCReturn();`

Reply:
Acknowledged. I took function [ConfigSetCheckSumOffload](https://github.com/OISF/suricata/pull/11997/files#:~:text=static%20int%20ConfigSetChecksumOffload(DPDKIfaceConfig%20*iconf%2C%20int%20entry_bool)) and [ConfigSetChecksumChecks](https://github.com/OISF/suricata/pull/11997/files#:~:text=static%20int%20ConfigSetChecksumChecks(DPDKIfaceConfig%20*iconf%2C%20int%20entry_bool)) as an example. Can you please point out the difference more clearly?

Reply:
```c
static void ConfigSetVlanStrip(DPDKIfaceConfig *iconf, int entry_bool)
{
    SCEnter();
    iconf->vlan_strip_enabled = entry_bool;
    SCReturn();
}
```